### ISSUE TITLE
PNG or MAT

### ISSUE THREAD

**Issue by PowerX-NF** [wfowler1/Unity3D-BSP-Importer#10]
do i have to change png to mat ? to import BSP

**Comment by wfowler1** [comment]
Nope. You just need to point the Texture Path to where the PNGs are stored. MAT files are automatically generated.
